autoware_utils/include/autoware_utils/ros/managed_transform_buffer.hpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2024 TIER IV, Inc.
+// Copyright 2025 The Autoware Contributors
```

Comment:
Copyright notices shouldn't be removed, the previous copyright still applies.

Reply:
The original file has been moved to `autoware_utils_pcl/managed_transform_buffer.hpp` and the copyright of that file has not changed. The new files were created by me this time for compatibility, so I think this year's copyright is appropriate.